utils/bm2rgbi/bm2rgbi/Generic565Converter.cs

```diff
@@ -5,100 +5,98 @@
 namespace bm2rgbi
 {
 
+  /// <summary>
+  /// generic converter for 565 format 64K colours
+  /// Format: RRRRRGGGGGGBBBBB (RGB)
+  /// Format: BBBBBGGGGGGRRRRR (BGR)
+  /// </summary>
+
+  public class Generic565Converter
+  {
+
     /// <summary>
-    /// generic converter for 565 format 64K colours
-    /// Format: RRRRRGGGGGGBBBBB (RGB)
-    /// Format: BBBBBGGGGGGRRRRR (BGR)
+    /// Convert to 565 format RGB
     /// </summary>
 
-    public class Generic565Converter
+    public void convertRGB(Bitmap bm, FileStream fs, Endianness ByteOrder)
```

Comment:
It's not changed. Just the formatting is not consistent in the original source and git thinks I have changed most of the source...